static/app/icons/iconRobot.tsx

```diff
@@ -0,0 +1,59 @@
+import {forwardRef} from 'react';
+
+import type {SVGIconProps} from './svgIcon';
+import {SvgIcon} from './svgIcon';
+
+const IconRobot = forwardRef<SVGSVGElement, SVGIconProps>((props, ref) => {
+  return (
+    <SvgIcon {...props} ref={ref}>
```

Comment:
Will likely want to pass this icon by the design team prior to shipping this but given that this will be behind a feature flag for a while, :shipit: